internal/export/client/init.go

```diff
@@ -55,15 +58,19 @@ func Init(conf ConfigurationStruct, l *zap.Logger) error {
 	var err error
 
 	// Create a database client
-	dbc, err = clients.NewDBClient(clients.DBConfiguration{
-		DbType:       clients.GetDatabaseType(conf.DBType),
+	dbc = mongo.NewClient(db.Configuration{
```

Comment:
You are hardcoding the db to mongo. Memorydb was also supported.

Reply:
Also Redis will be added